package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "eslint-config-logdna",
+  "version": "0.0.0",
+  "description": "LogDNA's preferred eslint config to be used across all projects",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "tap": "tap",
+    "pretest": "npm run lint",
+    "test": "tap"
+  },
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    }
+  },
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/logdna/eslint-config-logdna.git"
+  },
+  "author": {
+    "name": "LogDNA, Inc.",
+    "email": "help@logdna.com"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/logdna/eslint-config-logdna/issues"
+  },
+  "homepage": "https://github.com/logdna/eslint-config-logdna#readme",
+  "devDependencies": {
+    "eslint": "^7.9.0",
```

Comment:
Should we specify this as a `peerDependency`? https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

Reply:
Nah, I don't think we need that.  We've already had [a public complaint ](https://github.com/logdna/logger-node/issues/18)about it.  Nobody should really be using this but us, and I don't think we have to worry about eslint versions, etc.  Should just be able to use latest, and bump `eslint` as needed.

Can you

Reply:
ya, peer dependencies are a mess. better to avoid them. 
If anything we could put a note in the docs about our expected dependencies.

Reply:
Yep, it's already in the install instructions, but I'll add a sentence making it clear.